lib/pbench/server/api/__init__.py

```diff
@@ -142,41 +159,43 @@ def options(self):
 class GraphQL(Resource, CorsOptionsRequest):
     """GraphQL API for post request via server."""
 
-    def __init__(self):
-        CorsOptionsRequest.__init__(self, "Bad options request for the GraphQL query")
+    def __init__(self, app, api):
+        CorsOptionsRequest.__init__(
+            self, app, "Bad options request for the GraphQL query"
+        )
+        self.api = api
 
     def post(self):
-        global app
-        graphQL = (
-            f"http://{app.config_graphql.get('host')}:{app.config_graphql.get('port')}"
-        )
+        logger = self.app.logger
+        gql = self.app.config_graphql
+        graphQL = f"http://{gql.get('host')}:{gql.get('port')}"
         json_data = request.get_json(silent=True)
 
         if not json_data:
-            app.logger.warning("GraphQL: Invalid json object {}", request.url)
+            logger.warning("GraphQL: Invalid json object %s", request.url)
```

Comment:
Shouldn't this be `{}` still?

Reply:
This got pulled in by a merge, I think, and I missed the `%s`.